.dockerignore

```diff
@@ -1 +1,2 @@
+.git/*
```

Comment:
@Nuru these commands rely on 
`igt remote get-url` which doesn't work with out `.git`, which means it won't work in `geodesic`